src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -19,21 +19,20 @@ public void getStatistic(String fromFileName, String toFileName) {
         int sumOfSupply = 0;
         int sumOfBuy = 0;
         try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
-            String value = bufferedReader.readLine();
-            while (value != null) {
-                String[] arrayOfValues = value.split(REGEX);
-                if (arrayOfValues[OPERATION_TYPE_INDEX].equals(KEYWORD_SUPPLY)) {
-                    sumOfSupply += Integer.parseInt(arrayOfValues[AMOUNT_INDEX]);
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] statistic = line.split(REGEX);
+                if (statistic[OPERATION_TYPE_INDEX].equals(KEYWORD_SUPPLY)) {
+                    sumOfSupply += Integer.parseInt(statistic[AMOUNT_INDEX]);
                 } else {
```

Comment:
what if the file structure will change in the future? here you should look specifically for "buy"
```suggestion
                } else if (statistic[OPERATION_TYPE_INDEX].equals(KEYWORD_BUY) {
```